test_suite/slice.sugar.jsonnet

```diff
@@ -49,6 +49,14 @@ local arrCases = [
         output: arr,
     },
     {
+        input: arr[1::],
```

Comment:
Yeah there are several cases of that already, they should have a .fmt.golden file to hold the output of formatting those files (because they do not format to themselves).  If I accidentally broke the test, please fix it ;)